swagger-ui/spec.json

```diff
@@ -1,14 +1,7 @@
 {
-  "consumes": [
-    "application/json"
-  ],
-  "produces": [
-    "application/json"
-  ],
-  "schemes": [
-    "http",
-    "https"
-  ],
+  "consumes": ["application/json"],
```

Comment:
What's the differnces between this and merged.json? Does both needs to be commited?

Reply:
the `merged.json` contains also the ngalert endpoints
the need for this is that we do some [cleaning](https://github.com/grafana/grafana/blob/4f44ff7ce8117abadbeaa571bd2b9c64220e0dd4/pkg/services/ngalert/api/tooling/Makefile#L26) after generating the ngalert spec.

Reply:
Okay :+1: